server/odc-service/src/main/java/com/oceanbase/odc/service/structurecompare/StructureComparisonTask.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.odc.service.structurecompare;
+
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.InputStream;
+import java.util.List;
+import java.util.Objects;
+import java.util.concurrent.Callable;
+import java.util.stream.Collectors;
+
+import com.oceanbase.odc.core.shared.constant.DialectType;
+import com.oceanbase.odc.core.shared.constant.TaskStatus;
+import com.oceanbase.odc.metadb.structurecompare.StructureComparisonEntity;
+import com.oceanbase.odc.metadb.structurecompare.StructureComparisonRepository;
+import com.oceanbase.odc.metadb.structurecompare.StructureComparisonTaskRepository;
+import com.oceanbase.odc.service.flow.task.model.DBStructureComparisonTaskResult;
+import com.oceanbase.odc.service.flow.task.model.DBStructureComparisonTaskResult.Comparing;
+import com.oceanbase.odc.service.iam.model.User;
+import com.oceanbase.odc.service.iam.util.SecurityContextUtils;
+import com.oceanbase.odc.service.objectstorage.ObjectStorageFacade;
+import com.oceanbase.odc.service.objectstorage.model.ObjectMetadata;
+import com.oceanbase.odc.service.structurecompare.model.DBObjectComparisonResult;
+import com.oceanbase.odc.service.structurecompare.model.DBStructureComparisonConfig;
+import com.oceanbase.tools.dbbrowser.model.DBObjectType;
+
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * @author jingtian
+ * @date 2024/1/18
+ * @since ODC_release_4.2.4
+ */
+@Slf4j
+public class StructureComparisonTask implements Callable<DBStructureComparisonTaskResult> {
+    private DBStructureComparisonConfig srcConfig;
+    private DBStructureComparisonConfig tgtConfig;
+    private Long taskId;
+    private User user;
+    private StructureComparisonRepository structureComparisonRepository;
+    private StructureComparisonTaskRepository structureComparisonTaskRepository;
+    private ObjectStorageFacade objectStorageFacade;
+    private DefaultDBStructureComparator comparator = new DefaultDBStructureComparator();
+    @Getter
+    private volatile DBStructureComparisonTaskResult taskResult =
+            new DBStructureComparisonTaskResult(TaskStatus.PREPARING);
+    private static final String STRUCTURE_COMPARISON_RESULT_FILE_NAME = "structure-comparison-result.sql";
+    private final String STRUCTURE_COMPARISON_BUCKET_NAME = "structure-comparison";
```

Comment:
static?

Reply:
These two member variables do not need to be `static`, static is removed uniformly.